docs/pages/index.tsx

```diff
@@ -650,7 +650,6 @@ export const lists = createSchema({
               display: 'flex',
               gap: '1rem',
               justifyContent: 'center',
-              gap: '1rem',
```

Comment:
What's with you removing gaps? Is Safari somehow being silly?

Reply:
No gap here means it looks like this:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/1266923/123102444-69230980-d478-11eb-83f7-249045829e7e.png">

Reply:
It's a double property, see line 651

Reply:
nice find